docs/meetings/web/2021-11-03.md

```diff
@@ -0,0 +1,19 @@
+# Web meeting Nov 03 2021
+
+**Present:** Aapeli, Chad, Lucas, Ricardo, William
+
+## Discussion
+
+- Translations: all (?) strings now exist in json files, but app is still using the constants. Need to move across
```

Comment:
I'm thinking the incremental move to use the JSON files is a nice idea, but it also means we can't set up the translation components until we do the swap/check the naming of the translation key etc as discussed in my other PR, which means people who want to help us translate stuff won't really be able to help. Should we just bite the bullet and do it in large batches/all in one go? 🥲

Reply:
Yeah maybe, want to do half each?

Reply:
Yeah sounds like a plan!